### ISSUE TITLE
register button no response

### ISSUE THREAD

**Issue by v-dao** [tinode/chat#674]
**If you are not reporting a bug, please post to https://groups.google.com/d/forum/tinode instead.**
---

### Subject of the issue
Describe your issue here.

#### Server-side
- [x] web.tinode.co, api.tinode.co
- [ ] sandbox.tinode.co
- [ ] Your own setup:
  * platform (Windows, Mac, Linux etc)
  * version of Tinode server, e.g. `0.15.2-rc3`
  * database backend
  * cluster or standalone

#### Client-side
- [x] TinodeWeb/tinodejs: javascript client
  * Browser make and version.
- [ ] Tindroid: Android app
  * Android API level (e.g. 25).
  * Emulator or hardware, if hardware describe it.
- [ ] Tinodios: iOS app
  * iOS version
  * Simulator or hardware, if hardware describe it.
- [ ] tn-cli
  * Python version
- [ ] Chatbot
  * Python version
- Version of the client, e.g. `0.15.1`
- [ ] Your own client. Describe it:
  * Transport (gRPC, websocket, long polling)
  * Programming language.
  * gRPC version, if applicable.

### Steps to reproduce
I followed this tutorial to build the environment.
https://github.com/tinode/chat/blob/master/docker/README.md

```sh
$ docker network create tinode-net
$ docker run --name mysql --network tinode-net --restart always --env MYSQL_ALLOW_EMPTY_PASSWORD=yes -d mysql:5.7
$ docker run -p 6060:6060 -d --name tinode-srv --network tinode-net tinode/tinode-mysql:latest
```
![image](https://user-images.githubusercontent.com/47357234/124237722-68f6ce00-db4a-11eb-941a-dc1e25311bd2.png)
![image](https://user-images.githubusercontent.com/47357234/124237781-7ca23480-db4a-11eb-935d-1eca9fb7d3c2.png)

Tell us how to reproduce this issue.

### Expected behaviour
Tell us what should happen.

### Actual behaviour
Tell us what happens instead.

### Server-side log
Submission did not reach the server.

### Client-side log
![image](https://user-images.githubusercontent.com/47357234/124237804-8461d900-db4a-11eb-93cd-41b7b905e8fc.png)

**Comment by or-else** [comment]
The fix will be released in 0.17.6

**Comment by or-else** [comment]
Released. Closing.
